nixos/modules/services/hardware/fancontrol.nix

```
@@ -0,0 +1,39 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.fancontrol;
+
+in {
+
+  options.services.fancontrol = {
```

Maybe the hardware section is more appropriate?

```suggestion
  options.hardware.fancontrol = {
```

---

thanks, i wasn't aware of that section
what's the difference between nixos/modules/hardware and modules/services/hardware?